src/EditorFeatures/TestUtilities/Workspaces/TestWorkspace.cs

```diff
@@ -66,16 +67,16 @@ public TestWorkspace(ExportProvider exportProvider, string workspaceKind = null,
         }
 
         /// <summary>
-        /// Reset the thread affinity, in particular the designated foreground thread, to the active 
-        /// thread.  
```

Comment:
Although I don't mind that we're cleaning up the whitespace stuff here, can we make it a separate commit? Or do a big pass for the codebase first?